resources/fixtures/Stairville/Stairville-MH-100-Beam-36x3-LED.qxf

```diff
@@ -19,60 +19,60 @@
  <Channel Name="White" Preset="IntensityWhite"/>
  <Channel Name="Strobe">
   <Group Byte="0">Shutter</Group>
-  <Capability Min="0" Max="9">Off</Capability>
-  <Capability Min="10" Max="140">Increasing speed</Capability>
-  <Capability Min="141" Max="255">Random speed</Capability>
+  <Capability Min="0" Max="9" Preset="ShutterClose">Off</Capability>
```

Comment:
I'm not sure this is correct. I believe the manual intends "strobe off", which in this case means "ShutterOpen"

Reply:
Agreed and fixed.